src/index.html

```diff
@@ -1,11 +1,14 @@
 <!DOCTYPE html>
-<html>
+<html dir='ltr'>
   <head>
     <meta charset="utf-8">
+    <link rel="stylesheet" href="https://appsforoffice.microsoft.com/fabric/2.6.1/fabric.min.css"></link>
+    <script src="//cdnjs.cloudflare.com/ajax/libs/react/0.14.7/react.min.js"></script>
```

Comment:
Here's where i added the react dependency; also notice i added the minified version; you can remove the '.min' to get the non-minified, but be aware that there are performance differences between the two, especially in 15.x. I remember that the performance problems we saw in 15 were much less severe with the minified build.